cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -200,4 +200,33 @@ describe('Top toolbar tests.', function() {
 		cy.get('.leaflet-pane.leaflet-overlay-pane svg g.Graphic')
 			.should('exist');
 	});
+
+	it('Insert hyperlink.', function() {
+		cy.get('#toolbar-up .w2ui-scroll-right')
+		   .click();
+		
+		cy.get('#tb_editbar_item_link')
+			.should('be.visible').click();
```

Comment:
The test looks good to me. Just a small thing here: click() method implicitly contains a visibility check, so should('be.visible') part is unneeded, redundant.